setup.py

```diff
@@ -33,12 +33,14 @@
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6'
     ],
     install_requires=[
+        'future',
         'futures',
+        'six',
         'threadloop>=1,<2',
-        # we want thrift>=0.9.2.post1,<0.9.3, but we let the users pin to that
-        'thrift',
+        'thrift>=0.10.0',
```

Comment:
we need to keep the import without pinning to higher version, as it will make the library incompatible with many existing installations that may be using thrift <0.10. What's the reason to require 0.10, because PY3 doesn't work with 0.9? Is there a way to make this dependency sensitive to the Python version, i.e. only require 0.10 on Py3?